apps/web/src/hooks/useExperimentalFeatureEnabled.ts

```diff
@@ -0,0 +1,12 @@
+import { EXPERIMENTAL_FEATURES } from 'config/experminetalFeatures'
+import Cookies from 'js-cookie'
+import { useState } from 'react'
+
+const useExperimentalFeatureEnabled = (featureFlag: EXPERIMENTAL_FEATURES) => {
+  const isCookieInFeatureFlags = () => Cookies.get(`ctx-${featureFlag}`)?.toString() === 'true'
+  const [cookieValue] = useState<boolean>(isCookieInFeatureFlags())
+
+  return { featureEnabled: cookieValue }
```

Comment:
no need to wrap the value inside an object?
```suggestion
  return cookieValue
```